reana_server/decorators.py

```diff
@@ -72,7 +73,17 @@ def wrapper(*args, **kwargs):
         try:
             user = kwargs["user"]
             if user.has_exceeded_quota():
-                raise REANAQuotaExceededError()
+                quota = user.get_quota_usage()
+                message = "User quota exceeded.\n"
+                for resource in quota.keys():
+                    limit = quota[resource].get("limit", {}).get("raw", 0)
+                    usage = quota[resource].get("usage", {}).get("raw", 0)
+                    if limit > 0 and usage >= limit:
+                        resource_usage, _ = get_quota_resource_usage(
+                            quota[resource], "human_readable"
+                        )
+                        message += f"Resource: {resource}, usage: {resource_usage}\n"
```

Comment:
*suggestion:*

```python
for resource_type, resource in quota.items():
    limit = resource.get("limit", {}).get("raw", 0)
    usage = resource.get("usage", {}).get("raw", 0)
    if 0 < limit <= usage:
        resource_usage, _ = get_quota_resource_usage(
            resource, "human_readable"
        )
        message += f"Resource: {resource_type}, usage: {resource_usage}\n"
```

- Using `items()` to loop over both key and value
- Making `if` statement simpler